packages/apps/stock/src/i18n/fr.json

```diff
@@ -113,6 +113,9 @@
   "Stock_ToStockLocation": "Vers l'emplacement de stock",
   "Stock_DefineLines": "Compléter les lignes",
   "Stock_LinesAdded": "Lignes ajoutées",
```

Comment:
You forgot to remove translations that were used in the old screen like LinesAdded or DefineLines which are not used any more